src/createSingleLine.js

```diff
@@ -17,18 +17,26 @@ const createSingleLine = async (
   if (!client) throw new Error('client cannot be undefined');
   if (!modelId) throw new Error('Model ID cannot be undefined');
 
-  const validators = {};
+  let validators = {};
 
-  if (required) validators.required = {};
+  if (required || params.required) {
+    validators.required = {};
+    params.required = undefined;
+    required = undefined;
+  }
+  if (params.validators) {
+    validators = { ...params.validators, ...validators };
+    delete params.validators;
```

Comment:
Otherwise they get inline with `...params` and will be writen to the validation live above it.